examples/service-bindings/woof/src/index.ts

```diff
@@ -17,8 +17,8 @@ export class WoofWorker extends WorkerEntrypoint {
 
   // From Cloudflare docs: "Currently, entrypoints without a named handler are not supported"
   // TODO - Check if this is still the case that we need a fetch handler?
-  async fetch() {
-    return new Response(null, { status: 404 });
+  async fetch(request: Request | string) {
```

Comment:
I guess there's no linter checking this code, but if the parameter type is something important, maybe rename `request` to `_request`?

Reply:
yeah so you have to pass a string or Request otherwise the fetch call will fail. i will put an underscore!